lib/benchee/conversion/memory.ex

```diff
@@ -48,39 +48,48 @@ defmodule Benchee.Conversion.Memory do
 
   # Unit conversion functions
 
-  @type unit :: Unit.t
-  @type unit_atom :: atom
-  @type value_with_unit :: {number, unit}
+  @type unit_atom :: :byte | :kilobyte | :megabyte | :gigabyte | :terabyte
+  @type value_with_unit :: {number, Unit.t}
   @type value_with_unit_atom :: {number, unit_atom}
 
   @doc """
-  Converts a value for a specified unit and converts it to the equivalent of another unit of measure.
+  Converts a value for a specified unit atom and converts it to the equivalent of another unit of measure.
```

Comment:
also works with atoms, so we could say unit `(unit struct or atom representing the unit)`